circuit_tutorials/noir/pagerank/README.md

```diff
@@ -1,5 +1,6 @@
 # Pagerank Circuit
 
+# TODO: should package.json and package-lock.json be gitignored here?
```

Comment:
we want to keep package.json and package-lock.json in here - tells npm what dependencies to download to run the mjs script.  I'm about to push some readme edits that remove this and the analogous line in the gnark compression circuit